README.md

```diff
@@ -11,6 +11,8 @@ PHP WkHtmlToPdf provides a simple and clean interface to ease PDF and image crea
 [wkhtmltopdf](http://wkhtmltopdf.org). **The `wkhtmltopdf` and - optionally - `wkhtmltoimage`
 command must be installed and working on your system.** See the section below for details.
 
+For Windows usage make sure to add the path to wkhtmltopdf "bin" folder in the system variables PATH
```

Comment:
That's not really required. You can also set the path to the exe in the `binary` option without modifiying `PATH`. So we should recommand that first. How about:

> For Windows systems make sure to set the path to `wkhtmltopdf.exe` in the `binary` option. Alternatively you can add the directory to the system `PATH` variable.

Reply:
Adding it to the PATH will make the whhtmltopdf available as a command in CMD, so you will not need to make code changes when moving your code on a different system like Linux, it is no required unless you want your code to work crossed platform without additional changes. And something changed in the installer of wkhtmltopdf because from what I saw it used to add them directly when you installed it on windows but it does not do it anymore.

Reply:
Also, this will solve a lot of debugging for the initial "Failed without error" on Windows

Reply:
Well, it's a question of how you see it. I'd still find it easier to set the `binary` option instead of messing around with PATH. As long as we mention both, it should be OK. So if you update your PR I'll merge.

Reply:
Check now

Reply:
I'm sorry, but could you please remove that part about "moving to another system"? I want to keep the hint really short and not mention each and every possible combination of setups and pro/cons of each scenario. There are many pitfalls and we can't take care of them all. The developer stell needs to use his brain ;).